bodhi/server/validators.py

```diff
@@ -295,9 +295,10 @@ def validate_builds(request, **kwargs):
     for nvr in builds:
         build = request.db.query(Build).filter_by(nvr=nvr).first()
         if build and build.update is not None:
-            request.errors.add('body', 'builds',
-                               "Update for {} already exists".format(nvr))
-            return
+            if build.update.status is not UpdateStatus.unpushed:
```

Comment:
Values compared using 'is not' when equivalence is not the same as identity. Use '!=' instead.